src/cascadia/TerminalSettingsModel/CascadiaSettings.cpp

```diff
@@ -43,6 +43,15 @@ winrt::com_ptr<Profile> Model::implementation::CreateChild(const winrt::com_ptr<
     return profile;
 }
 
+std::string_view Model::implementation::LoadStringResource(int resourceID)
+{
+    const auto resource = FindResourceW(wil::GetModuleInstanceHandle(), MAKEINTRESOURCEW(resourceID), RT_RCDATA);

```

Comment:
You may still want to store the module handle in a local to make the code easier to read